src/cpu/x64/matmul/brgemm_matmul_copy_utils.cpp

```diff
@@ -396,10 +396,13 @@ void jit_brgemm_matmul_copy_a_impl_t<Vmm>::copy_K_loop(
         }
 
         // step 3: multiply by zp_b_val
-        mov(reg_zp_b_neg_val_ptr, ptr[param1 + GET_OFF(zp_b_neg_val_ptr)]);
-        const auto vmm_zp_b_neg_val = get_vmm_comp_acc(is_ymm_ ? 2 : 1);
-        uni_vbroadcastss(vmm_zp_b_neg_val, ptr[reg_zp_b_neg_val_ptr]);
-        uni_vpmulld(get_vmm_comp_acc(0), get_vmm_comp_acc(0), vmm_zp_b_neg_val);
+        if (conf_->wei_zp_type != brgemm_broadcast_t::per_n) {
```

Comment:
It should rather specify which zp_type it supports. Same applies to the code changed below.